ngx_rtmp_gop_cache_module.c

```diff
@@ -604,22 +629,24 @@ ngx_rtmp_gop_cache_send(ngx_rtmp_session_t *s)
             }
         }
 
-        if (meta == NULL && meta_version != gctx->meta_version) {
-            meta = handler->meta_message_pt(s, gctx->meta);
-            if (meta == NULL) {
-                ngx_rtmp_finalize_session(s);
-                return;
-            }
-        }
+        meta = NULL;
 
-        if (meta) {
-            meta_version = gctx->meta_version;
+        if (cache->meta) {
+            if (meta_version != cache->meta_version) {
+                meta = handler->meta_message_pt(s, cache->meta);
```

Comment:
merge two if into one
if (cache->meta  && meta_version != cache->meta_version)

Reply:
OK, thank you!